src/behaviors/filterable-store.js

```diff
@@ -0,0 +1,146 @@
+import angular from 'angular';
+import {Behavior} from './behavior';
+import {Handler as handlerDecorator} from '../store';
+
+import {
+  Transformer,
+  Transformable as transformableDecorator
+} from './transformable';
+
+import {
+  addBehavior,
+  camelcase,
+  Inject as injectDecorator
+} from '../utils';
+
+export class FilterableStoreBehavior extends Behavior {
+  constructor(instance, {collection} = {}) {
```

Comment:
It would be nice if for SortableStore, FilterableStore and PaginatableStore we could also configure the transformerWeight. Could easily be done by extracting transformerWeight from the config like we do with collection.